Source/Core/Common/MemArena.cpp

```diff
@@ -63,7 +63,7 @@ void MemArena::GrabSHMSegment(size_t size)
 #else
 	for (int i = 0; i < 10000; i++)
 	{
-		std::string file_name = StringFromFormat("dolphinmem.%d", i);
+		std::string file_name = StringFromFormat("/dolphinmem.%d", i);
```

Comment:
What's this slash for?
The SHM api doesn't use filenames, it shouldn't be needed